map/package.json

```diff
@@ -30,10 +30,10 @@
     "typescript": "~3.7.2"
   },
   "scripts": {
-    "start": "SKIP_PREFLIGHT_CHECK=true react-scripts start",
+    "start": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts start",
     "clean": "rm -rf build",
     "build-deps": "yarn --cwd ../model run build",
-    "build-map": "react-scripts build",
+    "build-map": "cross-env SKIP_PREFLIGHT_CHECK=true react-scripts build",
```

Comment:
what is the `cross-env` stuff

Reply:
Not a fan of it impacting our build pipeline. I think having it local development is fine, but not for prod use. Also our prod servers are linux based and don't have issues compiling the code. I think those on linux should make sure they set up their dev environment in a way that works for them without impacting the project as a whole. You can see the `.github/workflows` for ways on how to do that.

Basically we can't skip preflight on prod builds as that's meant to catch errors.